src/Platform/Microsoft.Testing.Platform/UI/StopwatchAbstraction.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Testing.Platform.UI;
```

Comment:
It seems it belongs to the `Helpers/System` folder, isn't it?

Reply:
The abstraction is only useful for testing this logger. I would not move it among System, unless we want to use it else where / everywhere.

Reply:
Could be globally useful but you are right, this is only used here for now.